src/pages/templates/blog/blog-article-list/articleList.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import {
+  Box,
+  Heading,
+  Link,
+  Image,
+  Text,
+  Divider,
+  HStack,
+  Tag,
+  Wrap,
+  WrapItem,
+  SpaceProps,
+  useColorModeValue,
+  Container,
+} from '@chakra-ui/react';
+
+interface IBlogTags {
+  tags: Array<string>;
+  marginTop?: SpaceProps['marginTop'];
+}
+
+const BlogTags: React.FC<IBlogTags> = (props) => {
+  return (
+    <HStack spacing={2} marginTop={props.marginTop}>
+      {props.tags.map((tag) => {
+        return (
+          <Tag size={'md'} variant="solid" colorScheme="orange" key={tag}>
+            {tag}
+          </Tag>
+        );
+      })}
+    </HStack>
+  );
+};
+
+interface BlogAuthorProps {
+  date: Date;
+  name: string;
+}
+
+export const BlogAuthor: React.FC<BlogAuthorProps> = (props) => {
+  return (
+    <Box marginTop="2" display="flex" alignItems="center">
+      <Image
+        borderRadius="full"
+        boxSize="40px"
+        src="https://100k-faces.glitch.me/random-image"
+        alt={`Avatar of ${props.name}`}
+      />
+      <Text marginLeft="2" fontWeight="medium">
+        {props.name}
+      </Text>
+      <Text marginLeft="2">—</Text>
+      <Text marginLeft="2">{props.date.toLocaleDateString()}</Text>
+    </Box>
+  );
+};
+
+const ArticleList = () => {
+  return (
+    <Container maxW={'7xl'} p="12">
+      <Heading as="h1">Stories by Chakra Templates</Heading>
+      <Box
+        marginTop={['1', '5']}
```

Comment:
I like the object syntax for responsive styles a little bit better: https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax